pkg/morph/client/multi.go

```diff
@@ -156,11 +185,12 @@ mainLoop:
 			return
 		case <-t.C:
 			c.switchLock.RLock()
+
```

Comment:
?

Reply:
could not find unlocking line while was preparing that PR so decide to chose a better (IMO) spacing

Reply:
a separate commit/keep as it was?

Reply:
I would keep if it then linter doesn't complain.